python/poetry.lock

```diff
@@ -976,13 +941,12 @@ test = ["coverage[toml] (>=5.2)", "hypothesis", "pytest (>=6.0)", "pytest-benchm
 
 [[package]]
 name = "pyspark"
-version = "3.5.4"
+version = "3.5.5"
```

Comment:
upgrades?

Reply:
We are relying on the latest one in CI, I just updated lock-file (by `poetry lock`)

Reply:
👍